sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md

```diff
@@ -1,33 +1,36 @@
-# Authentication Events Trigger for Azure Functions client library for .NET
+# Authentication events trigger for Azure Functions
```

Comment:
This will fail validation, as README must follow the required pattern for titles.  _([guideline](https://github.com/Azure/azure-sdk/blob/main/docs/policies/README-TEMPLATE.md))_

- [Example](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/README.md)
- [Example](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/README.md)

Reply:
Fixed.  Thanks, I'll read through the guidelines more closely.